sdk/personalizer/ai-personalizer-rest/rollup.config.js

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
you should be able to delete this file after updating the scripts in package.json